starport/cmd/chain_build.go

```diff
@@ -14,6 +14,7 @@ const (
 	flagRelease        = "release"
 	flagReleaseTargets = "release.targets"
 	flagReleasePrefix  = "release.prefix"
+	flagLDFlags        = "ldFlags"
```

Comment:
I suggest to move this to `config.yml` as that's where other build related customizations are made. 

By adding `LDFlags []string 'yaml:"ldflags"'` [here](https://github.com/tendermint/starport/blob/00adc1e/starport/chainconfig/config.go#L97).